test/integration/dashboardTest/submitNewLocationTest.js

```diff
@@ -0,0 +1,508 @@
+// Third-party dependencies
```

Comment:
Moved from `serverTest.js` and then made some changes that I'll try to point out.